README.md

```diff
@@ -40,11 +40,7 @@ add_filter( 'jetpack_tools_to_include', function( $tools ) {
 } );
 ```
 
-To make the bridge work, the site must have the ecommerce plan option set under the `at_options` option:
-
-```
-update_option( 'at_options', array( 'plan_slug' => 'ecommerce' ) );
-```
+To make bridge work, the site must have the eCommerce plan.
```

Comment:
Not sure how you test. Another option would be to simply create a small `Atomic_Plan_Manager` class with the needed method, and set a constant and make `current_plan_slug` always return it.

e.g.
```
class Atomic_Plan_Manager {
	const ECOMMERCE_PLAN_SLUG  = 'ecommerce';

	public static function current_plan_slug() {
		return self::ECOMMERCE_PLAN_SLUG;
	}
}

```